src/core/config/border_routing.h

```diff
@@ -117,4 +117,16 @@
 #define OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_BORDER_ROUTING_ADD_MOCK_PLAT_APIS
+ *
+ * Define to 1 to add mock (empty) implementation of infra-if platform APIs.
+ *
+ * This is intended for generating code size report only and should not be used otherwise.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_BORDER_ROUTING_ADD_MOCK_PLAT_APIS
+#define OPENTHREAD_CONFIG_BORDER_ROUTING_ADD_MOCK_PLAT_APIS 0
```

Comment:
nit: how about something like `OPENTHREAD_CONFIG_BORDER_ROUTING_MOCK_PLAT_APIS_ENABLE` for consistency with other binary configs?

Reply:
Updated them (also fixed the last one which said `ACK_MOCK_PLAT_API` instead of `ADD`) 😃